packages/model/src/user.ts

```diff
@@ -104,6 +104,7 @@ export interface OrcidWork {
 export interface UserTeam {
   id: string;
   displayName?: string;
+  teamInactiveSince?: string;
```

Comment:
Should this field be optional as a value is always set https://github.com/yldio/asap-hub/pull/2654/files#diff-9c6a0821d4e3970c62d1271d10fa7de6fe33823b99083824f6b340875adc5613R256

Reply:
I can be set as undefined

Reply:
If it can be set as undefined, shouldn't it be teamInactiveSince: string | undefined ?

Reply:
Afaik `teamInactiveSince?: string;` covers both scenarios. When the prop is not set and when it's set as undefined.